boggle-sveltexpress/svelte/src/App.svelte

```diff
@@ -14,6 +14,7 @@
       <Room id="{params.id}" />
     </Route>
     <Route path="/roombrowser" component="{RoomBrowser}"/>
+    <Route path="/game" component="{GamePage}"/>
```

Comment:
Integrate with the Room component instead? That way you'll have the sidebars for the players and chat aswell.